hieroplatform/src/test/java/org/hiero/sketch/SampleQuantileSketchTest.java

```diff
@@ -0,0 +1,62 @@
+package org.hiero.sketch;
+
+import org.hiero.dataset.ParallelDataSet;
+import org.hiero.sketches.*;
+import org.hiero.table.RecordOrder;
+import org.hiero.table.SmallTable;
+import org.hiero.table.api.ITable;
+import org.hiero.utils.RankInTable;
+import org.hiero.utils.TestTables;
+import org.junit.Test;
+
+import java.util.Arrays;
+
+public class SampleQuantileSketchTest {
+
+    @Test
+    public void SQSTest1() {
+        final int numCols = 2;
+        final int leftSize = 200000;
+        final int rightSize =200000;
+        final int resolution = 100;
+        final SmallTable leftTable = TestTables.getIntTable(leftSize, numCols);
+        RecordOrder rso = new RecordOrder();
+        for (String colName : leftTable.getSchema().getColumnNames()) {
+            rso.append(
+                    new ColumnSortOrientation(leftTable.getSchema().getDescription(colName), true));
+        }
+        final SampleQuantileSketch sqSketch = new
+                SampleQuantileSketch(rso, resolution, leftSize + rightSize);
+        final SampleList leftQ = sqSketch.create(leftTable);
+        RankInTable rLT = new RankInTable(leftTable, rso);
+        System.out.println(Arrays.toString(rLT.getRank(leftQ.getQuantiles(9))));
+        /*
```

Comment:
If you don't want this code you should just delete it.
Otherwise add a comment explaining why it is preserved.